### PR TITLE
fix: fixed memleaks and predefinedfilterscombobox and abstractlogview

### PR DESCRIPTION
**Pull request by GermanAizek**
fixed direct memleaks

### CODE REVIEW

File: src/ui/src/predefinedfilterscombobox.cpp [hunk 1]
```diff
@@ -102,6 +102,11 @@ PredefinedFiltersComboBox::PredefinedFiltersComboBox( QWidget* parent )
     setSizeAdjustPolicy( QComboBox::AdjustToContents );
 }
 
+PredefinedFiltersComboBox::~PredefinedFiltersComboBox()
+{
+    delete model_;
```
Comment (variar) [inline src/ui/src/predefinedfilterscombobox.cpp:107]: Setting this as a parent for the model object in PredefinedFiltersComboBox constructor is a more "Qt" way  to manage this without creating explicit destructors.

File: src/ui/src/predefinedfilterscombobox.cpp [hunk 2]
```diff
@@ -172,6 +177,7 @@ void PredefinedFiltersComboBox::setTitle( const QString& title )
 {
     auto* titleItem = new QStandardItem( title );
     model_->insertRow( 0, titleItem );
+    delete titleItem;
```
Comment (variar) [inline src/ui/src/predefinedfilterscombobox.cpp:180]: As far as I understand after insertRow call the ownership should be transferred to the model.

File: src/ui/src/abstractlogview.cpp [hunk 3]
```diff
@@ -536,9 +536,11 @@ AbstractLogView::~AbstractLogView()
     try {
         if ( quickFind_ ) {
             quickFind_->stopSearch();
+            delete quickFind_;
```
Comment (variar) [inline src/ui/src/abstractlogview.cpp:539]: Good find!

**Comment by GermanAizek** [comment]
@variar, oh my goh!
Thank you're alive. I was already going to make fork, but I couldn't devote time to it myself. Moreover, I would not be sure that I am correcting it correctly.

File: src/ui/src/abstractlogview.cpp (hunk 3 above)
Comment (GermanAizek) [reply src/ui/src/abstractlogview.cpp:539]: > Good find!

leave this change?

File: src/ui/src/predefinedfilterscombobox.cpp (hunk 2 above)
Comment (GermanAizek) [reply src/ui/src/predefinedfilterscombobox.cpp:180]: > As far as I understand after insertRow call the ownership should be transferred to the model.

leave this change?

File: src/ui/src/predefinedfilterscombobox.cpp (hunk 1 above)
Comment (GermanAizek) [reply src/ui/src/predefinedfilterscombobox.cpp:107]: > Setting this as a parent for the model object in PredefinedFiltersComboBox constructor is a more "Qt" way to manage this without creating explicit destructors.

leave this change?

**Pull request merged**